
{'date': 'Fri Aug 10 2018 10:12:31.216 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  if __name__ == '__main__':
  ^
IndentationError: expected an indented block
'''},
{'date': 'Fri Aug 10 2018 10:12:56.6 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  if __name__ == '__main__':
  ^
IndentationError: expected an indented block
'''},
{'date': 'Fri Aug 10 2018 10:14:07.181 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 11
  C9_OESTE= C9_SUL
  ^
IndentationError: expected an indented block
'''},
{'date': 'Fri Aug 10 2018 10:14:43.561 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 11
  c9_oeste= c0_sul
  ^
IndentationError: expected an indented block
'''},
{'date': 'Fri Aug 10 2018 10:14:50.376 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 11
  c9_oeste= c9_sul
  ^
IndentationError: expected an indented block
'''},
{'date': 'Fri Aug 10 2018 10:28:14.238 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 11
  c9_oeste= c9_sul
  ^
IndentationError: expected an indented block
'''},
{'date': 'Fri Aug 10 2018 10:48:36.99 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 94
    Museu()
  module <module> line 87
    Labirinto(sala_0, sala_1, sala_2, sala_3, sala_4, sala_5)
TypeError: __init__() takes 6 positional argument but more were given
'''},
{'date': 'Fri Aug 10 2018 11:23:56.878 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 87
  Labirinto(,sala_4,sala_5, sala_6,sala_7,sala_8)
             ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Aug 10 2018 11:41:41.329 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    STYLE["width"] = 850
AttributeError: 'module' object has no attribute '__setitem__'
'''},
{'date': 'Fri Aug 10 2018 11:41:42.384 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    STYLE["width"] = 850
AttributeError: 'module' object has no attribute '__setitem__'
'''},
{'date': 'Fri Aug 10 2018 11:44:58.696 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    STYLE["width"] = "850"
AttributeError: 'module' object has no attribute '__setitem__'
'''},
{'date': 'Mon Aug 13 2018 19:07:36.246 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 92
  MENSAGENS=[
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Aug 13 2018 19:07:43.812 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 92
  MENSAGENS=[
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Aug 13 2018 19:08:43.893 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 92
  MENSAGENS=[
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Aug 13 2018 19:10:48.24 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 92
  MENSAGENS=[
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Aug 20 2018 09:41:22.448 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module <string> line 93
  [sala_0.norte ,"primeiramente para usar a função labirinto do vitollino precisa ter definido as salas, depois se usa a função labirinto para 5 salas, ex: labirinto(salaA, sala B, sala C, salaD, sala E),  
                                                                                                                                                                                                              ^
SyntaxError: EOL while scanning string literal
'''},